Sources/SwiftDocC/Semantics/Symbol/Symbol.swift

```diff
@@ -509,6 +509,22 @@ extension Symbol {
             }
         }
     }
+    
+    /// Merge the different availability variants defined in the unified symbol,
+    /// and update the availability of the canonical symbol to consider all the different availability mixins instead of only the first one.
```

Comment:
I don't understand why the symbol semantic was only considering the first availability mixing before this. Can you help explain that?

Reply:
I can't really tell, aside from assuming that it went unnoticed.

From the `availability` declaration
```swift
/// The platforms on which the first variant of the symbol is available.
/// - note: Updating this property recalculates `isDeprecated`.
public var availability: SymbolGraph.Symbol.Availability? {
    get { availabilityVariants.firstValue }
    set { availabilityVariants.firstValue = newValue }
}
```
seems like the possibility of having different availability information in each of the unified symbol availability mixing wasn't considered.

If the test were always conducted using an Info.plist, it would be challenging to notice incorrect availability information since all platforms are displayed, but with incorrect versions in some cases.